disco_aws_automation/disco_elb.py

```diff
@@ -241,6 +243,8 @@ def get_or_create_elb(self, hostclass, security_groups, subnets, hosted_zone_nam
         elb_id = elb["LoadBalancerName"]
 
         self.route53.create_record(hosted_zone_name, cname, 'CNAME', elb['DNSName'])
+        if custom_cname:
+            self.route53.create_record(hosted_zone_name, custom_cname, 'CNAME', elb['DNSName'])
```

Comment:
That was the idea, yes.  Having the default one disappear seemed like a bad idea, especially since that might end up with having them both exist but point to different places.